Doc/library/os.rst

```diff
@@ -3665,16 +3665,16 @@ features:
 
    This example displays the number of bytes taken by non-directory files in each
    directory under the starting directory, except that it doesn't look under any
-   CVS subdirectory::
+   \_\_pycache\_\_ subdirectory::
 
       import os
       from os.path import join, getsize
       for root, dirs, files in os.walk('python/Lib/email'):
           print(root, "consumes", end=" ")
           print(sum(getsize(join(root, name)) for name in files), end=" ")
           print("bytes in", len(files), "non-directory files")
-          if 'CVS' in dirs:
-              dirs.remove('CVS')  # don't visit CVS directories
+          if '__pycache__' in dirs:
```

Comment:
You might want to update which directory is being walked over (currently it's python/Lib/email)

Reply:
Thank you for the suggestion! I’ll update the example to use `os.getcwd()` for better compatibility.

Reply:
Depending where you are actually running the command, it might be very very long for the user (say they run it from `$HOME`). And they could end up removing things they didn't want. Instead, we can walk over `'python/Lib'` instead.

Reply:
Thank you for the feedback! I’ve updated the example to use `python/Lib` as the base directory.

Reply:
What is the reason of this change?

Reply:
Since `__pycache__` is more common throughout `python/Lib` than `python/Lib/email`, I updated the directory to make the example more representative and practical.

Reply:
In addition, I only have a single `__pycache__` in my `email` package (even though email.mime exists) so you wouldn't see the "recursion" in this example I think if you compare with a `find` or a `ls`.

Reply:
Starting from `Lib` you will walk into `.git`, which may be non-desirable. `Lib/xml` contains several subdirectories with their own `__pycache__` directories.

Reply:
Ah maybe, `Lib/xml` is better then.